### DISCUSSION TITLE
How to use {count}gt again?

### DISCUSSION THREAD

**Discussion by PeterlitsZo** [LazyVim/LazyVim#356] [Q&A] [answered]
When I try to switch to the first tab, I prefer using `1gt`.

But in LazyVim, I cannot use this command to switch my tab. So how can I using the `{count}gt` command again please? Thanks a lot.

**Comment by dpetka2001** [comment, marked as answer]
`gt` is remapped to go to `Type definition`. I was also used in this keymap to navigate tabs so I have the following in my `lua/config/keymaps.lua`
```lua
-- Move to tabs convienently
map("n", "<leader>1", "1gt", { desc = "Move to tab 1" })
map("n", "<leader>2", "2gt", { desc = "Move to tab 2" })
map("n", "<leader>3", "3gt", { desc = "Move to tab 3" })
map("n", "<leader>4", "4gt", { desc = "Move to tab 4" })
map("n", "<leader>5", "5gt", { desc = "Move to tab 5" })
map("n", "<leader>6", "6gt", { desc = "Move to tab 6" })
map("n", "<leader>7", "7gt", { desc = "Move to tab 7" })
map("n", "<leader>8", "8gt", { desc = "Move to tab 8" })
map("n", "<leader>9", "9gt", { desc = "Move to tab 9" })
```
You could either try a similar approach, otherwise you would have to disable the mapping `gt` which you could do by doing something similar to [this](https://www.lazyvim.org/configuration/plugins#%EF%B8%8F-adding--disabling-plugin-keymaps).

**Reply by folke** [reply]
To add to this, you can disable the `gt` keymap as seen in the example for `K` here https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps

**Reply by dpetka2001** [reply]
Oh yes, it's an LSP mapping. Forgot about that and linked to the wrong reference in the site.

**Reply by PeterlitsZo** [reply]
Thanks a lot. :+1:
